### PR TITLE
changes for lang\ug.js

### PR DESCRIPTION
**Pull request by rider777**
there are many wrongs in this language file . For example,  translate "italic" to "Italian", “outdent” and "indent" is Increase the width and decrease the width, but I really don't know what this means by translation ，some of them are not translated . I've removed some unnecessary variable definitions. Also, I referred to the “UyghurSoft Dictionary”. In the end, I would like to thank @Ansar-786 , @blfrg and all Quasar Framework teams for their selfless dedication.

### CODE REVIEW

**Comment by Blfrg** [comment]
This is related to previous PR #7832 & [semi-resolved] issue #8123

Thank you @rider777 for taking the time to improve the generic Google translations with more accurate translations.
I verified the updated translations via Google (reverse translated) and they seem to convey the intended meanings.

LGTM :+1:

File: ui/lang/ug.js [hunk 1]
```diff
@@ -1,92 +1,88 @@
...
+    filter: 'سۈزگۈچ',
     refresh: 'يېڭىلاش'
   },
   date: {
-    days: days,
+    days: 'يەكشەنبە_دۈشەنبە_سەيشەنبە_چارشەنبە_پەيشەنبە_جۈمە_شەنبە_يەكشەنبە'.split('_'),
     daysShort: 'ي_د_س_چ_پ_ج_ش'.split('_'),
-    months: monthsShort,
-    monthsShort: monthsShort,
-    headerTitle: (date, model) => monthsShort[ model.month - 1 ] + 'نىڭ ' + model.day + '-كۈنى، ' + days[ date.getDay() ],
```
Comment (pdanpdan) [inline ui/lang/ug.js:28]: This language does not need a special processing to QDate headerTitle?

Comment (rider777) [reply ui/lang/ug.js:28]: > This language does not need a special processing to QDate headerTitle?

it was my mistake .
 of course need special processing. I am referring to the Arabic language file ， that doesn't have the headerTitle function ，I usually use qdate is also a custom title , I haven't had this problem .  I rewrote this function . 

headerTitle: function(date, model){
     return model.month + '-ئاينىڭ ' + model.day + '-كۈنى ، ' + this.days[ date.getDay() ]
    }

File: ui/lang/ug.js [hunk 2]
```diff
@@ -1,92 +1,88 @@
...
+    months: 'يانۋار(1)_فېۋرال(2)_مارت(3)_ئاپرىل(4)_ماي(5)_ئىيۇن(6)_ئىيۇل(7)_ئاۋغۇست (8)_سېنتەبىر(9)_ئۆكتەبىر(10)_نويابىر(11)_دېكابىر(12)'.split('_'),
+    monthsShort: '1-ئاي_2-ئاي_3-ئاي_4-ئاي_5-ئاي_6-ئاي_7-ئاي_8-ئاي_9-ئاي_10-ئاي_11-ئاي_12-ئاي'.split('_'),
     firstDayOfWeek: 1, // 0-6, 0 - Sunday, 1 Monday, ...
     format24h: false,
     pluralDay: 'كۈن'
```
Comment (rider777) [inline ui/lang/ug.js:27]: headerTitle: function(date, model){
     return model.month + '-ئاينىڭ ' + model.day + '-كۈنى ، ' + this.days[ date.getDay() ]
    }

File: ui/lang/ug.js [hunk 3]
```diff
@@ -1,92 +1,88 @@
...
-    days: days,
+    days: 'يەكشەنبە_دۈشەنبە_سەيشەنبە_چارشەنبە_پەيشەنبە_جۈمە_شەنبە_يەكشەنبە'.split('_'),
     daysShort: 'ي_د_س_چ_پ_ج_ش'.split('_'),
-    months: monthsShort,
-    monthsShort: monthsShort,
-    headerTitle: (date, model) => monthsShort[ model.month - 1 ] + 'نىڭ ' + model.day + '-كۈنى، ' + days[ date.getDay() ],
+    months: 'يانۋار(1)_فېۋرال(2)_مارت(3)_ئاپرىل(4)_ماي(5)_ئىيۇن(6)_ئىيۇل(7)_ئاۋغۇست (8)_سېنتەبىر(9)_ئۆكتەبىر(10)_نويابىر(11)_دېكابىر(12)'.split('_'),
+    monthsShort: '1-ئاي_2-ئاي_3-ئاي_4-ئاي_5-ئاي_6-ئاي_7-ئاي_8-ئاي_9-ئاي_10-ئاي_11-ئاي_12-ئاي'.split('_'),
     firstDayOfWeek: 1, // 0-6, 0 - Sunday, 1 Monday, ...
     format24h: false,
```
Comment (rider777) [inline ui/lang/ug.js:28]: headerTitle: function(date, model){
     return model.month + '-ئاينىڭ ' + model.day + '-كۈنى ، ' + this.days[ date.getDay() ]
    },

**Review by rider777 — COMMENTED** [review]
qdate

**Review by rider777 — COMMENTED** [review]
ok, that's completed.

**Comment by Blfrg** [comment]
@rider777 thank you for making the requested changes.
However I don't think this will get merged since you closed the PR.

@pdanpdan please consider re-opening this and reviewing further.

**Comment by rstoenescu** [comment]
Thanks for contributing!

**Comment by rstoenescu** [comment]
This will go into Quasar v2.6.0 and v1.18.6

**Comment by rider777** [comment]
> This will go into Quasar v2.6.0 and v1.18.6

@rstoenescu @pdanpdan @Blfrg 
Thanks for your support, I'm a new user of GitHub. I turned off pr by mistake before. Thank you to the quasar framework team for supporting our language. I love quasar, it's a great framework.

**Comment by rstoenescu** [comment]
@rider777 cheers!

**Pull request merged**